pkg/build/api/types.go

```diff
@@ -30,6 +30,9 @@ const (
 	// DropCapabilities is an environment variable that contains a list of capabilities to drop when
 	// executing a Source build
 	DropCapabilities = "DROP_CAPS"
+	// JenkinsDefaultServiceName is the default name of the Jenkins service deployed by
+	// JenkinsPipelineBuildStrategy.
+	JenkinsDefaultServiceName = "ci"
```

Comment:
@jstrachan `ci` was @smarterclayton suggestion... i agree with you that `jenkins-ci` would be a better name. Users might already have 'jenkins' server running, so maybe being explicit with `-ci` suffix isn't bad idea.